internal/mode/static/nginx/modules/test/vitest.config.ts

```diff
@@ -1,9 +1,9 @@
 import { defineConfig } from 'vitest/config';
 
 export default defineConfig({
-  test: {
-    coverage: {
-      reporter: ['text', 'json', 'html'],
-    },
-  },
+	test: {
+		coverage: {
+			reporter: ['text', 'json', 'html'],
+		},
+	},
```

Comment:
I'd just revert this file to what's on the main branch. Now it shows extra indentation.